workflow/controller/controller.go

```diff
@@ -729,7 +731,7 @@ func (wfc *WorkflowController) newPodInformer() cache.SharedIndexInformer {
 				if err != nil {
 					return
 				}
-				wfc.podQueue.Add(key)
+				wfc.podQueue.AddAfter(key, enoughTimeForInformerSync)
```

Comment:
Is there any reason we need to add 1s delay here? We already have 1s sleep on reprocessing workflow.

Reply:
I wanted to make sure there was time to sync the informer, but of course at this point, we have must already synced the informer.  I don't think we should every re-queue with less that 1 second delay.